api-tester/src/main/java/com/revenuecat/apitester/kotlin/DeprecatedPurchasesAPI.kt

```diff
@@ -1,129 +1,12 @@
-            override fun onError(error: PurchasesError, userCancelled: Boolean) {}
-        }
-        val makePurchaseListener = object : MakePurchaseListener {
-            override fun onCompleted(purchase: Purchase, purchaserInfo: PurchaserInfo) {}
-            override fun onError(error: PurchasesError, userCancelled: Boolean) {}
-        }
-        val receiveCustomerInfoListener = object : ReceivePurchaserInfoListener {
-            override fun onReceived(purchaserInfo: PurchaserInfo) {}
-            override fun onError(error: PurchasesError) {}
-        }
-        purchases.getOfferings(receiveOfferingsListener)
-        purchases.getSubscriptionSkus(skus, skusResponseListener)
-        purchases.getNonSubscriptionSkus(skus, skusResponseListener)
-
-        val productChangeCallback = object : ProductChangeCallback {
-            override fun onCompleted(storeTransaction: StoreTransaction?, customerInfo: CustomerInfo) {}
-            override fun onError(error: PurchasesError, userCancelled: Boolean) {}
-        }
-        val purchaseCallback = object : PurchaseCallback {
-            override fun onCompleted(storeTransaction: StoreTransaction, customerInfo: CustomerInfo) {}
-            override fun onError(error: PurchasesError, userCancelled: Boolean) {}
-        }
-        purchases.purchaseProduct(activity, skuDetails, upgradeInfo, productChangeListener)
-        purchases.purchaseProduct(activity, storeProduct, upgradeInfo, productChangeListener)
-        purchases.purchaseProduct(activity, skuDetails, upgradeInfo, productChangeCallback)
-        purchases.purchaseProduct(activity, skuDetails, makePurchaseListener)
-        purchases.purchaseProduct(activity, storeProduct, makePurchaseListener)
-        purchases.purchaseProduct(activity, skuDetails, purchaseCallback)
-        purchases.purchasePackage(activity, packageToPurchase, upgradeInfo, productChangeListener)
-        purchases.purchasePackage(activity, packageToPurchase, makePurchaseListener)
-        purchases.restorePurchases(receiveCustomerInfoListener)
-        purchases.logOut(receiveCustomerInfoListener)
-        purchases.getPurchaserInfo(receiveCustomerInfoListener)
-        purchases.getPurchaserInfo(object : ReceiveCustomerInfoCallback {
-            override fun onReceived(customerInfo: CustomerInfo) {}
-            override fun onError(error: PurchasesError) {}
-        })
-        purchases.getCustomerInfo(receiveCustomerInfoListener)
-        purchases.invalidatePurchaserInfoCache()
-        purchases.removeUpdatedPurchaserInfoListener()
-
-        val updatedPurchaserInfoListener: UpdatedPurchaserInfoListener? = purchases.updatedPurchaserInfoListener
-        purchases.updatedPurchaserInfoListener = object : UpdatedPurchaserInfoListener {
-            override fun onReceived(purchaserInfo: PurchaserInfo) {
-            }
-        }
-
         purchases.allowSharingPlayStoreAccount = true
```

Comment:
I left it because we deprecated it in v3, and kept it in v4 because it's used when posting the receipt. I think we can make it private and default it to `identityManager.currentUserIsAnonymous()`, so people stop setting it, but it's something we should discuss